.ado/publish.yml

```diff
@@ -5,38 +5,156 @@
 variables:
   Ver.Major: 0
   Ver.Minor: 0
-  Ver.Patch: 0
+  Ver.Patch: 4
```

Comment:
This shows the latest published version.
The versions up to and including this one have been published during the pipeline testing.

After the transition to ESRP Publishing, the UNpublishing requires a separate discussion with ESRP team. So, for now I did not remove/UNpublished the versions published during the pipleine testing.